doc/api/crypto.md

```diff
@@ -1030,6 +1030,7 @@ const fs = require('fs');
 const hash = crypto.createHash('sha256');
 
 const input = fs.createReadStream('test.js');
+hash.setEncoding('hex');
 input.pipe(hash).pipe(process.stdout);
```

Comment:
This is completely optional, but maybe this way it becomes a bit more obvious where the hex encoding takes place? 

```suggestion
input.pipe(hash).setEncoding('hex').pipe(process.stdout);
```

Reply:
Yes, that would be great to have because it gives the more apparent view where the hex encoding is being done.

Reply:
Yep that would work, do you want me to submit another commit with your suggestion? Thank you!

Reply:
@vbarzana You can just add that commit to this PR, or use the “Commit suggestion” button here in the Github interface :)

Reply:
Done! Thank you, didn't see the button cause I was on the mobile 😉